src/math/Ray.js

```diff
@@ -67,19 +72,26 @@ Object.assign( Ray.prototype, {
 
 	}(),
 
-	closestPointToPoint: function ( point, optionalTarget ) {
+	closestPointToPoint: function ( point, target ) {
+
+		if ( target === undefined ) {
+
+			console.warn( 'THREE.Ray: .closestPointToPoint() target is now required' );
```

Comment:
@WestLangley 

This triggers in the Points.raycast() testpoint function.

(I haven't looked at other objects raycast methods)

Reply:
see #13530

Reply:
Thanks @aardgoose Trying to track these down...